docker/.env.example

```diff
@@ -9,7 +9,7 @@ SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJzZX
 # Auth
 
 ## General
-SITE_URL=http://localhost:3000
+SITE_URL=http://localhost:3001
```

Comment:
I also changed the default site URL's port to 3001, since Studio runs on port 3000 by default as well.